contentgrid-spring-boot-autoconfigure/src/main/java/com/contentgrid/spring/boot/autoconfigure/data/web/ContentGridSpringDataRestAutoConfiguration.java

```diff
@@ -32,7 +34,8 @@
                 // is initialized so @ConditionalOnBean works correctly. Putting the annotation directly on that class
                 // does not work, because it is not an autoconfiguration, but is initialized directly when this parent class
                 // is initialized.
-                "internal.org.springframework.content.rest.boot.autoconfigure.ContentRestAutoConfiguration"
+                "internal.org.springframework.content.rest.boot.autoconfigure.ContentRestAutoConfiguration",
+                "org.springframework.boot.autoconfigure.data.rest.RepositoryRestMvcAutoConfiguration"
```

Comment:
Since this one is on the classpath, you could reference it using a class (that helps against accidental typos)

Reply:
I can't have two `@AutoConfigureAfter`s on the same class.

Reply:
No, but I wonder if you can set both `value` and `name` in an `@AutoconfigureAfter`:

```java
@AutoconfigureAfter(
   name={
       "internal.org.springframework..."
   },
   value={
      RepositoryRestMvcAutoConfiguration.class
   }
)
```